src/sentry/constants.py

```diff
@@ -714,6 +714,8 @@ class InsightModules(Enum):
 UPTIME_AUTODETECTION = True
 TARGET_SAMPLE_RATE_DEFAULT = 1.0
 SAMPLING_MODE_DEFAULT = "organization"
+ROLLBACK_ENABLED_DEFAULT = True
```

Comment:
these don't _really_ need to be two constants, having just `ROLLBACK_DEFAULT = True` would do the same thing because of how things are set up